### DISCUSSION TITLE
Price movement to the right/left of the curve

### DISCUSSION THREAD

**Discussion by hosseinnedaee** [Jeiwan/uniswapv3-book#47] [Milestone 3. Cross-tick Swaps]
The text is saying "If we buy all available amount of ETH from this range, the range will contain only the other token, USDC, and the price will move to the right of the curve." but I think it moves to the left of the curve, Am I wrong? If yes, Why I can't get it?!

**Comment by Jeiwan** [comment]
While it sounds confusing, it's in fact correct: the price will move to the right **of the curve**. Meaning, it'll exit the price range of the curve and will move to another price range. This is illustrated in the next chapter: https://uniswapv3book.com/docs/milestone_3/cross-tick-swaps/#how-cross-tick-swaps-work
![](https://uniswapv3book.com/images/milestone_3/price_range_dynamics.png)

**Reply by ValyaZay** [reply]
Sorry, but I'm lost here too. If the price of eth moves to the right of the curve, the eth becomes cheaper? Please elaborate on this a bit

**Reply by Jeiwan** [reply]
If it moves to the right **of** the curve, it becomes more expensive. If it moves to the right **on** the curve, it becomes cheaper. The price axis is an instrument that helps arrange liquidity ranges (the curves on the image). The movement of the price within a liquidity range (within a curve) is defined by the CMM formula: `x * y = k`. I.e. the bigger the value of an axis the cheaper the asset denoted by the axis. Since, in Uniswap V3, liquidity ranges are bounded, the price can reach one of the axes, meaning that a liquidity range might not have enough liquidity. When this happens, a liquidity range starting at a higher price (to the right of the liquidity range) will be found.

**Reply by OoXooOx** [reply]
You are really confuse people. You must announce that for price range you will use other diagram, because in picture from 1-st and 2-nd milestone price go lower to right and get bigger when move to left from P current

**Reply by vishwa7dcap786** [reply]
> While it sounds confusing, it's in fact correct: the price will move to the right **of the curve**. Meaning, it'll exit the price range of the curve and will move to another price range. This is illustrated in the next chapter: https://uniswapv3book.com/docs/milestone_3/cross-tick-swaps/#how-cross-tick-swaps-work ![](https://camo.githubusercontent.com/a2b4b10fe09b48446908976c0ea0ea8d8c6112c0360d2c2407a7bf43d44ccace/68747470733a2f2f756e69737761707633626f6f6b2e636f6d2f696d616765732f6d696c6573746f6e655f332f70726963655f72616e67655f64796e616d6963732e706e67)

It is right. Consider the current curve b-5500,c-5000,a-4545 .  If we buy all available amount of ETH from this range, the range will contain only the other token, USDC. It moves to the left on the current curve and exit the current price range and it also seems like now it is in the right of the new price range of the new curve. so It moves to the right of the another price range and exits the current price range by moving left.
